Buildings/Controls/OBC/FDE/PackagedRTUs/BAScontroller.mo

```diff
@@ -0,0 +1,309 @@
+  annotation (Dialog(group="System and building parameters"));
+
+  parameter Real minSATset(
+  final unit="K",
+  final displayUnit="degC",
+  final quantity="ThermodynamicTemperature")=14
+  "Minimum supply air temperature reset value"
+  annotation (Dialog(tab="Supply air temperature", group="Temperature limits"));
+
+  parameter Real maxSATset(
+  final unit="K",
+  final displayUnit="degC",
+  final quantity="ThermodynamicTemperature")=19
+  "Maximum supply air temperature reset value"
+  annotation (Dialog(tab="Supply air temperature", group="Temperature limits"));
+
+  parameter Real HeatSet(
+  final unit="K",
+  final displayUnit="degC",
+  final quantity="ThermodynamicTemperature")=35
+  "Setback heating supply air temperature set point"
+  annotation (Dialog(tab="Supply air temperature", group="Temperature limits"));
+
+  parameter Real maxDDSPset(
+  final unit="Pa",
+  final displayUnit="Pa",
+  final quantity="PressureDifference")=500
+  "Maximum down duct static pressure reset value"
+  annotation (Dialog(tab="Supply fan", group="Down duct pressure limits"));
+
+  parameter Real minDDSPset(
+  final unit="Pa",
+  final displayUnit="Pa",
+  final quantity="PressureDifference")=125
+  "Minimum down duct static pressure reset value"
+  annotation (Dialog(tab="Supply fan", group="Down duct pressure limits"));
+
+  // ---inputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput occ
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{-140,60},{-100,100}}),
+        iconTransformation(extent={{-140,60},{-100,100}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput occReq
+    "Terminal unit occupancy requests"
+    annotation (Placement(transformation(extent={{-140,34},{-100,74}}),
+        iconTransformation(extent={{-140,34},{-100,74}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput sbcReq
+    "Terminal unit setback cooling requests"
+    annotation (Placement(transformation(extent={{-140,8},{-100,48}}),
+        iconTransformation(extent={{-140,8},{-100,48}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput sbhReq
+    "Terminal unit setback heating requests"
+    annotation (Placement(
+        transformation(extent={{-140,-20},{-100,20}}), iconTransformation(
+          extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput totCoolReqs
+    "Total terminal unit cooling requests"
+    annotation (Placement(
+        transformation(extent={{-140,-48},{-100,-8}}), iconTransformation(
+          extent={{-140,-48},{-100,-8}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput highSpaceT
+    "Highest space temperature reported from all terminal units"
+    annotation (Placement(transformation(extent={{-140,-74},{-100,-34}}),
+        iconTransformation(extent={{-140,-74},{-100,-34}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput mostOpenDam
+    "Most open damper position of all terminal units" annotation (
```

Comment:
Is this maximum open position? if yes, change the name to `maxDamPos`.

Reply:
No, this is the most open damper position, not a maximum damper position.